_unittest/test_00_EDB.py

```diff
@@ -2936,14 +2994,54 @@ def test_147_find_dc_shorts(self):
         assert len(edbapp.nets["DDR4_DM3"].find_dc_short()) == 0
         edbapp.close()
 
-    def test_148_load_amat(self):
-        assert "Rogers RO3003 (tm)" in self.edbapp.materials.materials_in_aedt
-        material_file = os.path.join(self.edbapp.materials.syslib, "Materials.amat")
-        assert self.edbapp.materials.add_material_from_aedt("Arnold_Magnetics_N28AH_-40C")
-        assert "Arnold_Magnetics_N28AH_-40C" in self.edbapp.materials.materials.keys()
-        assert self.edbapp.materials.load_amat(material_file)
+    @patch.object(builtins, "open", new_callable=mock_open, read_data=MATERIALS)
+    def test_149_materials_read_materials(self, mock_file_open):
+        """Read materials from an amat file."""
+        materials = Materials(MagicMock(materials=["copper"]))
+        expected_res = {
+            "Polyflon CuFlon (tm)": {"permittivity": 2.1, "tangent_delta": 0.00045},
+            "Water(@360K)": {
+                "thermal_conductivity": 0.6743,
+                "mass_density": 967.4,
+                "specific_heat": 4206,
+                "thermal_expansion_coeffcient": 0.0006979,
+            },
+        }
+        mats = materials.read_materials("some path")
+        assert mats == expected_res
+
+    def test_150_material_load_syslib_amat(self):
+        """Load material from an amat file."""
```

Comment:
File extension should be in all capital letters. Please do a search for "amat" and change to "AMAT" instead.

Reply:
Sure, I'll go that :+1: